lib/extensions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const path = require('path');
+
+module.exports = {
+
+  withConflictExclusions: function(BaseGenerator) {
+    return class extends BaseGenerator {
+      constructor(args, opts) {
+        super(args, opts);
+
+        this._conflictExclusions = [];
+        this._extendConflicter();
+      }
+
+      _updateFileContent(filePath, pipeline, options) {
+        const { force } = options || {};
```

Comment:
Please const { force } = options || { force : false}; for better code safety and less further double checks